src/__experimental__/components/checkbox/checkbox-group.component.js

```diff
@@ -2,42 +2,73 @@ import React from "react";
 import PropTypes from "prop-types";
 import tagComponent from "../../../utils/helpers/tags";
 import { StyledCheckboxGroup } from "./checkbox.style";
-import FormField from "../form-field";
-import { InputGroupBehaviour } from "../../../__internal__/input-behaviour";
+import Fieldset from "../../../__internal__/fieldset";
 
 const CheckboxGroup = (props) => {
-  const { children, groupName, error, warning, info } = props;
+  const {
+    children,
+    groupName,
+    legend,
+    error,
+    warning,
+    info,
+    mb,
+    required,
+    legendInline,
+    legendWidth,
+    legendAlign,
+    legendSpacing,
+  } = props;
 
   const groupLabelId = `${groupName}-label`;
 
   return (
-    <InputGroupBehaviour>
+    <Fieldset
+      role="group"
+      legend={legend}
+      inline={legendInline}
+      legendWidth={legendWidth}
+      legendAlign={legendAlign}
+      legendSpacing={legendSpacing}
+      error={error}
+      warning={warning}
+      info={info}
+      mb={mb}
+      isRequired={required}
+      {...tagComponent("checkboxgroup", props)}
+    >
       <StyledCheckboxGroup
         aria-labelledby={groupLabelId}
```

Comment:
I see that you didn't change the line but I can't find related `ID` to this `aria-labelledby` so probably it doesn't work as it should

Reply:
removed as it's already labelled in a fieldset